scripts/scil_compute_fodf_metrics.py

```diff
@@ -198,9 +185,19 @@ def main():
     if args.afd_sum:
         save(afd_sum, affine, args.afd_sum)
 
+    if args.gfa:
+        nib.save(nib.Nifti1Image(gfa_map, affine), args.gfa)
+
+    if args.qa:
+        nib.save(nib.Nifti1Image(qa_map, affine), args.qa)
+
+    if args.rgb:
+        nib.save(nib.Nifti1Image(rgb_map.astype('uint8'), affine), args.rgb)
+
     if args.peaks:
-        nib.save(nib.Nifti1Image(reshape_peaks_for_visualization(peaks_dirs),
+        nib.save(nib.Nifti1Image(reshape_peaks_for_visualization(peak_dirs),
                                  affine), args.peaks)
+
     if args.visu:
```

Comment:
@jchoude Is it still relevant to have this --visu option ?

I would rather see that disappear and use `nib.save()` everywhere

Reply:
Yeurk fibernavigator... I agree we should save it but with an extension saying it's RGB friendly or something like that.